pkg/disk/partition_table_test.go

```diff
@@ -1221,3 +1120,1013 @@ func TestAddPartitionsForBootMode(t *testing.T) {
+							LogicalVolumes: []disk.LVMLogicalVolume{
+								{
+									Name: "varloglv",
+									Size: 12 * datasizes.MiB, // rounded up to next extent (4 MiB)
+									Payload: &disk.Filesystem{
+										Label:        "var-log",
+										Type:         "xfs",
+										Mountpoint:   "/var/log",
+										FSTabOptions: "defaults",
+										UUID:         "fb180daf-48a7-4ee0-b10d-394651850fd4",
+									},
+								},
+								{
+									Name: "rootlv",
+									Size: 52 * datasizes.MiB, // rounded up to the next extent (4 MiB)
+									Payload: &disk.Filesystem{
+										Label:        "root",
+										Type:         "xfs",
+										Mountpoint:   "/",
+										FSTabOptions: "defaults",
+										UUID:         "a178892e-e285-4ce1-9114-55780875d64e",
+									},
+								},
+								{
+									Name: "datalv",
+									Size: 100 * datasizes.MiB,
+									Payload: &disk.Filesystem{
+										Label:        "data",
+										Type:         "ext4", // the defaultType
+										Mountpoint:   "/data",
+										FSTabOptions: "defaults",
+										UUID:         "e2d3d0d0-de6b-48f9-b44c-e85ff044c6b1",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		"lvm-multivg": {
+			// two volume groups, both unnamed, and no root lv defined
+			// NOTE: this is currently not supported by customizations but the
+			// PT creation function can handle it
+			customizations: &blueprint.DiskCustomization{
+				Partitions: []blueprint.PartitionCustomization{
+					{
+						Type:    "lvm",
+						MinSize: 100 * datasizes.MiB,
+						VGCustomization: blueprint.VGCustomization{
+							LogicalVolumes: []blueprint.LVCustomization{
+								{
+									Name:    "varloglv",
+									MinSize: 10 * datasizes.MiB,
+									FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+										Mountpoint: "/var/log",
+										Label:      "var-log",
+										FSType:     "xfs",
+									},
+								},
+							},
+						},
+					},
+					{
+						Type: "lvm",
+						VGCustomization: blueprint.VGCustomization{
+							LogicalVolumes: []blueprint.LVCustomization{
+								{ // unnamed + untyped logical volume
+									MinSize: 100 * datasizes.MiB,
+									FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+										Mountpoint: "/data",
+										Label:      "data",
+										FSType:     "ext4", // TODO: remove when we reintroduce the default fs
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			options: &disk.CustomPartitionTableOptions{
+				DefaultFSType:    disk.FS_EXT4,
+				BootMode:         platform.BOOT_HYBRID,
+				RequiredMinSizes: map[string]uint64{"/": 3 * datasizes.GiB},
+			},
+			expected: &disk.PartitionTable{
+				Type: disk.PT_GPT, // default when unspecified
+				UUID: "0194fdc2-fa2f-4cc0-81d3-ff12045b73c8",
+				Size: 818*datasizes.MiB + 16*datasizes.MiB + 3*datasizes.GiB + datasizes.MiB, // start + size of last partition (VG) + footer
```

Comment:
(nitpick) same as above, it becomes a bit tricky to follow where the 818 comes from, the comment seems to be a bit terse, so *maybe* a comment like `// start + efi + boot + 2 VGs + rootsize` or something might slightly more helpful